tests/models/cohere/test_modeling_cohere.py

```diff
@@ -291,6 +291,14 @@ def setUp(self):
         self.model_tester = CohereModelTester(self)
         self.config_tester = ConfigTester(self, config_class=CohereConfig, hidden_size=37)
 
+    @unittest.skip("foo")
+    def test_initialization(self):
+        super().test_initialization()
+
+    @unittest.skip("foo")
+    def test_fast_init_context_manager(self):
+        super().test_fast_init_context_manager()
```

Comment:
I will open issue to keep this track, but let's follow what we have discussed earlier that we sometimes also need to skip some tests.